cypress/fixtures/b2b-mp/yves/product/dynamic-publish-and-synchronize-dms.json

```diff
@@ -0,0 +1,112 @@
+{
+  "data": {
+    "type": "dynamic-fixtures",
+    "attributes": {
+      "synchronize": true,
+      "operations": [
+        {
+          "type": "transfer",
+          "name": "LocaleTransfer",
+          "key": "locale",
+          "arguments": { "id_locale": 66, "locale_name": "en_US" }
+        },
+        {
+          "type": "helper",
+          "name": "haveStore",
+          "key": "store",
+          "arguments": [{ "name": "TEST_STORE" }]
```

Comment:
This part looks wrong.
Imagine I wonna execute this test first in a chain and I expect that new store will be created via the backoffice.

In MP context it's a bit complicated:
- Login as backoffice user
- Create store
- Create product
- Login as merchant user
- Create merchant product
- Trigger P&S

In B2B, B2C, SUITE repositories please use concrete products.
In B2B-MP, B2C-MP repositories use merchant products.